data/products/sapcal/sle15/sp4/packages.yaml

```diff
@@ -1,2 +1,9 @@
 packages:
   _namespace_sapcal_libopenssl_old: Null
+  _namespace_sapcal_extra:
+    package:
+      - xmlstarlet
+      - libcap-progs
+      - openssl
```

Comment:
openssl is already in the image. We need to verify requests.

Reply:
I didn't find it mentioned in any of the SAPCAL related packages.yaml files, therefore I added it explicitly.